README.md

```diff
@@ -21,11 +21,11 @@ docker compose build && docker compose create && docker compose start
 docker restart shop-prestashop-1
 ```
 
-### Zbudowanie nowego obrazu kontenera sklepu
+### Zbudowanie nowego obrazu kontenera sklepu, przy odpalaniu uważać na zakończenia linii w plikach pre-install.sh itp. - powinny być Unixowe, ale git może pobrać jako Windowsowe
```

Comment:
Fajnie, że to zauważyłeś, ale w takim razie lepiej zmienić Dockerfile tak, aby zamieniał CRLF na LF. 

```Dockerfile
RUN apt install -y do2unix # wcześniej razem z innymi instalacjami
# Poniżej kod wykona konwersję dla wszystkich plików w katalogach (typ pliku to zwykły plik) i każdy plik pójdzie do konwersji
RUN find /tmp/init-scripts /tmp/post-install-scripts /tmp/pre-install-scripts -type f | xargs dos2unix 
```

Wtedy dobrze będzie to usunąć też z dokumentacji - większa automatyzacja procesu

Reply:
Można też do dokumentacji dodać informację o uruchamianiu kodu w C#. Ewentualnie też przenieść go katalog wyżej, tak aby nie było katalogu w katalogu, gdzie w katalogu product-init jest tylko katalog z projektem w C#

Reply:
Dockerfile zmieniony, w readme dodana informacja o C# i usunięty jeden z katalogów